elementQuery.js

```diff
@@ -164,6 +164,12 @@
                             processSelector(rule[cssRules][j].selectorText);
                         }
                     }
+                    // Handle @imported stylesheets
+                    else if (rule.styleSheet && rule.styleSheet.cssRules && rule.styleSheet.cssRules.length) {
```

Comment:
Possibly... to be honest this was a quick hack in order to evaluate whether this script would even work in our environment. In our case Drupal just spews out a bunch of imports in a <style> block and this seems to work.